.github/workflows/test.yml

```diff
@@ -87,6 +87,12 @@ jobs:
         with:
           name: Test Results
           path: "**/e2e-tests/test-results"
```

Comment:
I think db-test results are already uploaded here, since `./gradlew e2eTest` calls `:db-tests:e2eTest`, but you might want to double check i.e. remove this additional `upload-artifact` call and see if db-test results are still there.

Reply:
I'm not seeing the `db-tests` results in the `e2e-tests` directory

Reply:
Will verify without just to see

Reply:
Yeah the additional upload artifact is needed to grab the nested `e2eTest` folder from the `db-tests`

Reply:
Yep you are correct!